Classes/Widgets/Provider/QueueSizeDataProvider.php

```diff
@@ -19,6 +17,8 @@
  * The TYPO3 project - inspiring people to share!
  */
 
+namespace FriendsOfCrawler\CrawlerWidget\Widgets\Provider;
+
```

Comment:
I prefer Namespace over License, but I can live with it ;) Preferences shouldn't be discussed. as there is selden reasoning behind :)

Reply:
php-cs-fixer does this automatically, I haven't found an option to change that.